Source/Initialization/WarpXAMReXInit.cpp

```diff
@@ -111,6 +137,7 @@ namespace {
         apply_workaround_for_warpx_numprocs();
         set_device_synchronization();
         override_default_tiling_option_for_particles();
+        add_constants();
```

Comment:
Maybe call this first before the other routines. It is unlikely, but those read in some parameters that could possibly depend on the constants.